### PR TITLE
cut() → cut

### PR DESCRIPTION
**Pull request by plotnick**
PR checklist:

- [x] Added changelog entry.

### CODE REVIEW

File: docs/changelogs/vNEXT.rst [hunk 1]
```diff
@@ -26,6 +26,12 @@ To improve readability of Polar policies, the ``isa`` operator has
 been replaced with ``matches`` (:ref:`operator-matches`). Any policies using the
 ``isa`` operator will need to be migrated.
 
+cut() predicate replaced with cut keyword
+-----------------------------------------
+
+The Polar "cut" operator was previously parsed like a predicate, but
+it isn't one. The new syntax ``cut`` (without the empty parentheses)
+emphasizes its role as a keyword.
```
Comment (gj) [inline docs/changelogs/vNEXT.rst:34]: as an operator?

File: docs/examples/inheritance/07-common-cut.polar [hunk 2]
```diff
@@ -23,7 +23,7 @@ exception(actor: Actor, "read", resource: Lab) if
 
 # Not actually sure if this works... needs some testing. If it doesn't work
 # we should think if there is some way to incorporate something like this, because
-# cut() is not ideal if it is common. (Although not sure if the idea of
+# cut is not ideal if it is common. (Although not sure if the idea of
 # exception is much better). Instead we almost want an ALL operator which
 # would change the matching for a given predicate call, ensuring that all
 # match (instead of just one) before returning a true result.
```
Comment (gj) [inline docs/examples/inheritance/07-common-cut.polar:29]: This entire comment block, from L24-32, looks like it's in need of an update. Doesn't have to be part of this PR.

File: docs/using/polar-syntax.rst [hunk 3]
```diff
@@ -285,17 +285,17 @@ will compare the value of the variable age with 10 and unify if it's less than 1
...
 
-The *cut* operator, which in Polar is written as ``cut()``, commits the query
-engine to the enclosing rule definition, and refuses to consider any others. Any
-definitions that have already run are not "un-run", though, or avoided by using
-cut; it just ensures that no *others* will run. Such "other" rule definitions
-are often less specific rules, and the use of `cut()` can be used, e.g., to
-override an ``allow`` rule for a less-specific class.
-
-``cut()`` can appear anywhere in a rule body, but terms that
+The ``cut`` operator the query engine to the enclosing rule definition,
```
Comment (gj) [inline docs/using/polar-syntax.rst:288]: Missing a word?

File: docs/using/polar-syntax.rst [hunk 4]
```diff
@@ -285,17 +285,17 @@ will compare the value of the variable age with 10 and unify if it's less than 1
...
+and refuses to consider any others. Rules that have already run are not
+"un-run", though, or avoided by using cut; it just ensures that no
+*others* will run. Such "other" rule definitions are often less specific
```
Comment (gj) [inline docs/using/polar-syntax.rst:291]: Would love to add a phrase underscoring _why_ the "other" rules are less specific, e.g.:
> Due to the way in which oso sorts rules for evaluation, such "other" rule definitions...

File: docs/changelogs/vNEXT.rst (hunk 1 above)
Comment (plotnick) [reply docs/changelogs/vNEXT.rst:34]: Right.

File: docs/examples/inheritance/07-common-cut.polar (hunk 2 above)
Comment (plotnick) [reply docs/examples/inheritance/07-common-cut.polar:29]: Glancing at it, I would say that the entire file needs an update. It's supposed to be about "Policies where cut is common", but it doesn't have a single `cut`. But we have plenty of bigger fish to fry today.

Comment (dhatch) [reply docs/examples/inheritance/07-common-cut.polar:29]: This file isn't used in the documentation yet.

File: docs/using/polar-syntax.rst (hunk 4 above)
Comment (plotnick) [reply docs/using/polar-syntax.rst]: OK, fell into it a little and added more than a phrase.

File: docs/using/polar-syntax.rst [hunk 5]
```diff
@@ -285,25 +285,35 @@ will compare the value of the variable age with 10 and unify if it's less than 1
...
+the enclosing rule definition: the query engine will not run any others.
+Rule definitions that have already run are not "un-run", though, or avoided
+by using cut; it just ensures that no *others* will run.
+
+Because Polar runs rules in most-to-least-specific order, these "other"
+rule definitions are always *less specific* than the current one; i.e.,
+they may have specializers that are superclasses (and therefore less specific)
+of those in the current rule. This allows ``cut`` to override a rule that
+is specialized on a less specific class. You can think of ``cut`` as a sort
+of dual to ``super()`` in other object-oriented languages (e.g., Python):
```
Comment (gj) [inline docs/using/polar-syntax.rst:300]:
```suggestion
of dual to ``super()`` in object-oriented languages (e.g., Python):
```

Comment (plotnick) [reply docs/using/polar-syntax.rst:300]: No, Polar is an object-oriented language.

Comment (gj) [reply docs/using/polar-syntax.rst:300]: you're an object-oriented language

**Pull request merged**